addons/survey/views/survey_templates.xml

```diff
@@ -421,139 +419,146 @@
     <template id="question_simple_choice" name="Question: simple choice">
         <t t-set="answer_line" t-value="answer_lines.filtered(lambda line: line.suggested_answer_id)"/>
         <t t-set="comment_line" t-value="answer_lines.filtered(lambda line: line.value_char_box)"/>
-        <div class="row o_survey_form_choice"
+        <t t-set="answers_contains_image" t-value="question.suggested_answer_ids.filtered(lambda x : x.value_image)"/>
+        <t t-set="use_half_columns" t-value="survey.questions_layout == 'page_per_question' and answers_contains_image"/>
+        <div class="row g-2 o_survey_form_choice"
              t-att-data-name="question.id"
              data-question-type="simple_choice_radio">
             <t t-set="item_idx" t-value="0"/>
-            <div t-attf-class="col-lg-12 d-flex flex-wrap">
-                <t t-set="has_correct_answer" t-value="scoring_display_correction and any(label.is_correct for label in question.suggested_answer_ids)"/>
-                <t t-foreach='question.suggested_answer_ids' t-as='label'>
-                    <t t-set="item_idx" t-value="label_index"/>
-                    <t t-set="answer_selected" t-value="answer_line and answer_line.suggested_answer_id.id == label.id"/>
-                    <t t-set="is_correct" t-value="label.is_correct"/>
-
-                    <!--Used for print mode with corrections -->
-                    <t t-set="answer_class" t-if="not has_correct_answer" t-value="''" />
-                    <t t-set="answer_class" t-elif="is_correct" t-value="'bg-success'" />
-                    <t t-set="answer_class" t-elif="not is_correct" t-value="'bg-danger'" />
-
+            <t t-set="has_correct_answer" t-value="scoring_display_correction and any(label.is_correct for label in question.suggested_answer_ids)"/>
+            <t t-foreach='question.suggested_answer_ids' t-as='label'>
+                <t t-set="answers_contains_image" t-value="question.suggested_answer_ids.filtered(lambda x : x.value_image)"/>
+                <t t-set="item_idx" t-value="label_index"/>
+                <t t-set="answer_selected" t-value="answer_line and answer_line.suggested_answer_id.id == label.id"/>
+                <t t-set="is_correct" t-value="label.is_correct"/>
```

Comment:
I'm not convinced setting this variable is very useful. Especially with the suggestion below, using `label.is_correct` would work fine there and at the other occurrence(s).